### PR TITLE
OSDOCS-9543: Updating GA and EOL dates for 4.15

### PR DESCRIPTION
**Pull request by eohartman**
Version(s):
4.15

Issue:
[OSDOCS-9543](https://issues.redhat.com/browse/OSDOCS-9543)

Link to docs preview:
[OSD Life cycle dates](https://71662--ocpdocs-pr.netlify.app/openshift-dedicated/latest/osd_architecture/osd_policy/osd-life-cycle#sd-life-cycle-dates_osd-life-cycle)
[ROSA Classic Life Cycle dates](https://71662--ocpdocs-pr.netlify.app/openshift-rosa/latest/rosa_architecture/rosa_policy_service_definition/rosa-life-cycle#sd-life-cycle-dates_rosa-life-cycle)
[ROSA HCP Life cycle dates](https://71662--ocpdocs-pr.netlify.app/openshift-rosa/latest/rosa_architecture/rosa_policy_service_definition/rosa-hcp-life-cycle#sd-life-cycle-dates_rosa-hcp-life-cycle)

QE review:
- [ ] QE has approved this change.

### CODE REVIEW

File: modules/rosa-policy-disaster-recovery.adoc
Comment (mjlshen) [inline modules/rosa-policy-disaster-recovery.adoc:1]: The intended changes look good, but I think this change was mistakenly bundled in :D

Comment (eohartman) [reply modules/rosa-policy-disaster-recovery.adoc:1]: Yup, I'm fixing that now! Thank you!

**Review by mjlshen — APPROVED** [review]
/lgtm

**Comment by xueli181114** [comment]
LGTM

**Review by mjlshen — COMMENTED** [review]
/lgtm cancel

My fault - I mistakenly suggested the GA date for 2023, but it's 2024!

**Review by mjlshen — APPROVED** [review]
/lgtm

**Pull request closed without merging**